src/screen/show_refs.rs

```diff
@@ -18,50 +18,40 @@ pub(crate) fn create(config: Rc<Config>, repo: Rc<Repository>, size: Rect) -> Re
         size,
         Box::new(move || {
             let style = &config.style;
-            let head = repo.head().ok();
 
             Ok(iter::once(Item {
-                id: "branches".into(),
+                id: "local_branches".into(),
                 display: Line::styled("Branches".to_string(), &style.section_header),
                 section: true,
                 depth: 0,
                 ..Default::default()
             })
+            .chain(branches(&repo, Some(BranchType::Local), &style.branch)?)
+            .chain(iter::once(Item {
+                id: "remote_branches".into(),
+                display: Line::styled("Remote".to_string(), &style.section_header),
```

Comment:
In magit, there's one section per remote
```
Remote FrancoRav (git@github.com:FrancoRav/gitu.git) (4)…
Remote golden-expiriensu (git@github.com:golden-expiriensu/gitu.git) (6)…
Remote jondysinger (git@github.com:jondysinger/gitu.git) (2)
  feat/confirm-quit               test: dont test quit confirmation for now (off by default)…
  master                          ci: set 'release' environment for release workflow…

```

It doesn't need to look the same (maybe in the future?), but would it make more sense to label this "Remote branches" for now?

Reply:
Yeah, it would be very cool to have them separated, lemme try

Reply:
Done